streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedDemo.java

```diff
@@ -168,7 +168,7 @@ public PageViewByRegion apply(PageView view, UserProfile profile) {
                     }
                 })
                 .groupByKey(Serdes.String(), pageViewByRegionSerde)
-                .count(TimeWindows.of("GeoPageViewsWindow", 7 * 24 * 60 * 60 * 1000L).advanceBy(1000))
+                .count(TimeWindows.of(7 * 24 * 60 * 60 * 1000L).advanceBy(1000), "GeoPageViewsWindow")
```

Comment:
Given that this is one of the few included examples of Streams, can we come up with a more descriptive name than "GeoPageViewsWindow"?  For one, the singular "Window" suggests that there's only one such window or whatever.  For example, "WeeklyPageViewsByRegion"?